src/anyio/_core/_eventloop.py

```diff
@@ -152,8 +170,16 @@ def claim_worker_thread(
 
 
 def get_async_backend(asynclib_name: str | None = None) -> type[AsyncBackend]:
+    global forced_backend
+
     if asynclib_name is None:
-        asynclib_name = sniffio.current_async_library()
+        if os.getenv("PYTEST_CURRENT_TEST"):
```

Comment:
not `ANYIO_BACKEND`?

Reply:
Ah...this must be a leftover from my alternate solutions. I'll see if I can improve this further.

Reply:
Actually I don't think it should be `ANYIO_BACKEND` either. You'll figure it out :)

Reply:
I think I got it right this time around.